### ISSUE TITLE
Question: Can this be solved by including our own realpath implementation?

### ISSUE THREAD

**Issue by jordwalke** [jordwalke/esy#54]
See this line here:
https://github.com/jordwalke/esy/blob/master/.bin/esy#L5

Can our own provided `realpath` implementation work correctly instead of this snippet?

@yunxing @andreypopp

**Comment by chenglou** [comment]
Doesn't our realpath compiles C++?

**Comment by jordwalke** [comment]
Yes, but we can easily compile it as a dependency. It's like one line of code to generate it.
